docs/processes/post_mortem.md

```diff
@@ -0,0 +1,124 @@
+
+- **Presenter:** Incident Manager or a designated team member.
+- **Content:**
+  - What happened?
+  - When did it happen?
+  - Impact on users and systems.
+
+### 3. Timeline Review (15 minutes)
+
+- **Objective:** Go through the sequence of events from detection to resolution.
+- **Content:**
+  - Detection: How was the incident discovered?
+  - Response: Key actions taken and decisions made.
+  - Resolution: Steps that led to resolving the issue.
+
+### 4. Root Cause Analysis (20 minutes)
+
+- **Method:** Use the "Five Whys" or other root cause analysis techniques.
+- **Objective:** Identify the underlying causes, not just the symptoms.
+
+### 5. What Went Well (10 minutes)
+
+- **Discussion:** Acknowledge effective actions and strategies.
+- **Purpose:** Highlight positive aspects to reinforce good practices.
+
+### 6. What Could Be Improved (20 minutes)
+
+- **Discussion:** Identify gaps in processes, tools, or communication.
+- **Focus Areas:**
+  - Processes that need refinement.
+  - Tools or systems that need enhancement.
+  - Training or documentation gaps.
+
+### 7. Action Items (15 minutes)
+
+- **Objective:** Define specific tasks to prevent recurrence.
+- **Examples:**
+  - Update or create playbooks.
+  - Improve monitoring and alerting systems.
+  - Update documentation.
+  - Update tests.
+  - Schedule training sessions.
+
+### 8. Assign Responsibilities (5 minutes)
+
+- **Task:** Assign owners and set due dates for each action item.
+- **Tracking:** Add action items to the backlog in Dev Ops with appropriate priority.
+
+### 9. Closing Remarks (5 minutes)
+
+- **Facilitator:** Summarize key takeaways.
+- **Next Steps:** Outline how the findings will be communicated and implemented.
+
+## Guidelines
+
+- **Blameless Environment:** Focus on processes and systems, not individuals.
+- **Open Communication:** Encourage honest and constructive feedback.
+- **Documentation:** Record all findings and action items in the post-mortem report.
+- **Follow-Up:** Ensure action items are tracked to completion.
+
+## Post-Mortem Report Template
+
+After the meeting, compile a report using the following structure:
+
+```markdown
+# Incident name
+
+> Date and time of the incident
+
+### Incident Summary
+
+- Brief description of what happened.
+- Impact analysis.
+
+### Timeline of Events
+
+- Detailed sequence from detection to resolution.
+
+### Root Cause(s)
+
+- Primary and contributing factors.
+
+### What Went Well
+
+- Effective actions and strategies.
+
+### What Could Be Improved
+
+- Identified gaps and areas for improvement.
+
+### Action Items
+
+- List of tasks with assigned owners and due dates.
+
+### Additional Notes
+
+- Any other relevant information.
+```
+
+## Storage and Accessibility
+
+- Save the post-mortem report in the shared repository (e.g., GitHub Wiki or `docs/post_mortems` folder). (XXX: TBD where we would like to store this)
+- Ensure it is accessible to all team members for future reference.
+
```

Comment:
I wonder if this Post mortem report could be stored in Azure as well for easier access together with any related PBIs/ tasks that occurred along the way?

Reply:
Yeah, I don't find Azure to be the easiest thing to navigate, but maybe indeed that would be the best place to store it (and we can probably generate a link that will filter by what we want to see). Curious to hear other's opinions on this too.

Reply:
Let's keep DevOps as single source of truth and not create separate documents/repos.
Fine to fill in a Word template and attach that to the DevOps item if that is believed to be better.
In DevOps one can easily create Queries. If that does not suffice, then a separate document with a list of links is fine. Preferably in Teams then.